i18n/locales/da-DK/hourofcode/activity-guidelines.md

```diff
@@ -8,9 +8,9 @@ title: Guidelines for Øvelser
   }
 </style>
 
-{{ signup_button }}
+{{ tilmeld_knap }}
```

Comment:
@castro-jorge looks like a translator in Danish translated a bunch of variable names, which is going to break most of hourofcode.com in that language

Reply:
oh, actually it looks like we set this up to fail nicely when a bad template name is given, so this should be okay to ship until we get it fixed; it just means the pages will render without the signup button.